config/webpack.main.config.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path');
+
+const extendConfig = require('./webpack.shared');
+
+
+module.exports = extendConfig(() => {
+    return {
+        entry: [
+            path.resolve(__dirname, '../app/main.js')
+        ],
+        output: {
+            path: path.resolve(__dirname, '../build'),
+            filename: 'main.js'
+        },
+        target: 'electron-main',
```

Comment:
Scratch that, I missed that this is now a 4th webpack config file... I thought this was the `shared`.